setup.py

```diff
@@ -80,6 +80,7 @@ def get_version():
         'Framework :: Django :: 3.2',
         'Framework :: Django :: 4.0',
         'Framework :: Django :: 4.1',
+        'Framework :: Django :: 4.2',
         'Operating System :: POSIX',
         'Programming Language :: Python :: 3.6',
```

Comment:
3.6 support was dropped earlier, I think this classifier is not needed anymore.

Reply:
Done